init-tools.cmd

```diff
@@ -52,10 +59,17 @@ if NOT exist "%DOTNET_LOCAL_PATH%" (
 
 :afterdotnetrestore
 
+echo Restoring component %COMPONENT%...
```

Comment:
Instead of creating a separate component restore please just add the package to the init-tools.msbuild script. We don't need to do this as a separate step.